### DISCUSSION TITLE
Two-factor authentication (2FA)

### DISCUSSION THREAD

**Discussion by gettheboom** [chromiecraft/chromiecraft#3289] [Suggestions]
UPDATED:

Chromiecraft has this feature in-game, just not enabled by default. If you'd like to benefit of this feature, please watch this short clip on how to enable 2FA for your account --- > https://youtu.be/Qty1jcAlj4E  <---

Hi,

For those who are not familiar with two-factor authentication (2FA), it's like a second password, an extra layer of protection for your account.
Basically, when you are entering your credentials (username and password), you won't be able to gain access to your account until you can confirm a PIN either sent to your personal email/SMS/or any 2-nd factor auth apps like google. 
ChromieCraft has some sort of extra layer of security, however, it only works to lock your account and log in only from ONE single IP address. That puts you at risk of being locked out of your account if you have a dynamic IP address which is most likely what your internet provider has nowadays.

Not sure if it helps, but:
[Github 2FA](https://github.com/azerothcore/azerothcore-wotlk/commit/1320cc21cfe4557cdd7a03b83d24439cc4deab3c)

Please make it a priority as being secured it's a must.

**Comment by Protandina** [comment]
Would have to agree. If hard to implement for in-game, at least make it so that if selected in options it would require  2FA for site login.
Also is it possible if 2FA implemented for site login to have an option to freeze account so that in-game login is not possible.

**Comment by gettheboom** [comment]
@Protandina looks like they do have it enabled in-game, I have made a short tutorial on how to enable 2FA 

https://youtu.be/Qty1jcAlj4E
